Library/ENV/4.3/cc

```diff
@@ -276,6 +281,11 @@ class Cmd
     %W[#{sysroot}/usr/lib /usr/local/lib]
   end
 
+  def keg_path
+    return nil if formula.nil?
+    "#{cellar}/#{formula}/#{formula_version}"
```

Comment:
I don't think so, since this is a standalone script and not running under `brew`, so we don't have `Formula` objects. Or do you mean the syntax for joining with slashes? I think this form is pretty readable for paths.